wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/Angular2CalculatorTest.wiki

```diff
@@ -0,0 +1,28 @@
+Sample test on [[Angular][https://angular.io/]] application. 
+
+!note '''Please note:''' The application used in this test does NOT seem to work with !-PhantomJs-!. It does work when executed with Chrome or Firefox.
+
+!note Due to the nature of [[Plunker][https://plnkr.co/]] which hosts the application in an iframe, the page must first be opened with !-BrowserTest-!, switch to the embedded iframe and wait for the application to load.
+
+
+|table template|add                      |
+|enter         |@{first} |as |id=first   |
+|enter         |@{second}|as |id=second  |
+|select        |+        |for|id=operator|
+|click         |Go!                      |
```

Comment:
It would be even better if the first and second had their own placeholder and the operator an aria-label attribute. Then we could use the placeholders and the aria-label, instead of those ugly ids in the table template....